target-data/time-series.csv

```diff
@@ -19689,1494 +19689,1494 @@
 "2025-02-18","Queens","ILI ED visits","2022-09-24",686
 "2025-02-18","Staten Island","ILI ED visits","2022-09-24",53
 "2025-02-18","Unknown","ILI ED visits","2022-09-24",54
+"2025-02-18","San Antonio","Flu ED visits pct","2022-10-01",0.78
```

Comment:
Why are there changes to the time-series data as of 2025-02-18, shouldn't this file just have updates as_of 2025-03-04?